Packs/Venafi/Integrations/VenafiV2/VenafiV2.yml

```diff
@@ -1,309 +1,309 @@
 category: IT Services
 commonfields:
-  id: Venafi TLS Protect
+  id: VenafiTLSProtect
   version: -1
 configuration:
-  - defaultvalue: "https://ao-tlspd.dev.ven-eco.com"
-    display: Server URL (e.g., https://192.168.0.1)
-    name: server
-    required: true
-    type: 0
-    section: Connect
-  - displaypassword: Password
-    name: credentials
-    required: true
-    type: 9
-    display: User Name
-    section: Connect
-  - display: Client ID
-    name: client_id
-    required: true
-    type: 0
-    section: Connect
-  - display: Trust any certificate (not secure)
-    name: insecure
-    type: 8
-    required: false
-  - display: Use system proxy settings
-    name: proxy
-    type: 8
-    required: false
+- defaultvalue: "https://ao-tlspd.dev.ven-eco.com"
+  display: Server URL (e.g., https://192.168.0.1)
+  name: server
+  required: true
+  type: 0
+  section: Connect
+- displaypassword: Password
+  name: credentials
+  required: true
+  type: 9
+  display: User Name
+  section: Connect
+- display: Client ID
+  name: client_id
+  required: true
+  type: 0
+  section: Connect
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
 description: Retrieves information about certificates stored in Venafi.
 display: Venafi TLS Protect
-name: Venafi TLS Protect
+name: VenafiTLSProtect
```

Comment:
Lets revert to be with spaces, this is the name, it can be with spaces, and leave the id as is without them.